src/index.js

```diff
@@ -1,30 +1,70 @@
 import './style.css';
 
 const list = document.querySelector('.list');
-const tasks = [
-  {
-    description: 'Learn coding',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Do yoga',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'Have dinner with family',
-    completed: false,
-    index: 3,
-  },
-];
+const desc = document.querySelector('.description');
+const form = document.querySelector('.toDoForm');
+let toDos = JSON.parse(localStorage.getItem('toDos')) || [];
 
-function addList(tasks) {
+const addToDo = () => {
+  form.addEventListener('submit', () => {
+    const toDo = {
+      description: desc.value,
+      completed: false,
+      index: toDos.length + 1,
```

Comment:
New tasks should have the property `completed` set to `false` and the property `index` set to the value of the new array length 🚀